core/utilities/mcmlln-tools.rb

```diff
@@ -52,7 +53,7 @@ def self.makeDir(directory)
 
     # An array listing everything in a directory
     def self.dirList(directory)
-      Dir.entries(directory)
+      Dir.entries(directory) - ['..', '.']
```

Comment:
What does this shorthand mean? Can you add a comment to explain what the fallback is for?